src/neural/onnx/converter.cc

```diff
@@ -329,7 +411,13 @@ std::string Converter::MakeEncoderLayer(
                                           {embedding_size, dff_size}, {1, 0}));
   flow = builder->Add(name + "/ffn/dense1/b", flow,
                       *GetWeghtsConverter(layer.ffn.dense1_b, {dff_size}));
-  flow = MakeActivation(builder, flow, name + "/ffn/dense1", activation);
+  if (layer.mha.has_smolgen) {
```

Comment:
There's also a separate `square_relu_ffn: true` configuration parameter in the yaml file for this (which I also missed out in the proto update). `has_smolgen` can be used as a proxy for now though until all of that is settled.